README.md

```diff
@@ -1,2 +1,25 @@
 # cmssw_vscode
 Recipies to develop CMSSW using VSCode
+
+## Indexing files
+Indexing files allows VSCode to easily resolve references to imported files.
```

Comment:
It would be nice to present in form of screenshot of better animated gif what this tool does.

Readers may not realise that it may allow the vscode user to jump to the source code of some class which is cloned in current session. That should be clearly written here. 

Also some link to vscode indexing feature documentation may be helpful here.

<img width="893" alt="Screenshot 2024-04-27 at 17 09 44" src="https://github.com/CTPPS/cmssw_vscode/assets/7374733/6aae1997-6144-47e9-9b55-ef8029255948">

Reply:
Done.